server/src/main/java/org/sonarsource/ldap/server/ApacheDS.java

```diff
@@ -1,6 +1,6 @@
 /*
  * SonarQube LDAP Test Server
- * Copyright (C) 2009-2017 SonarSource SA
+ * Copyright (C) 2009-2018 SonarSource SA
```

Comment:
@julienlancelot maybe it's time for `2019` ? 😉

Reply:
Good catch, I've probably not use the latest pom version...

Reply:
@Godin It should be better now :)